source/use-cases/client-side-field-level-encryption-guide.txt

```diff
@@ -623,23 +612,22 @@ MedcoMD engineers write a function to create a new patient record:
 
       .. code-block:: javascript
 
-         function insertPatient(collection, name, ssn, bloodType, medicalRecords,  policyNumber, provider) {
-             collection.insertOne({
-                name: name,
-                ssn: ssn,
-                bloodType: bloodType,
-                medicalRecords: medicalRecords,
-                insurance: {
-                    policyNumber: policyNumber,
-                    provider: provider
-                }
-             })
-             .then((writeResult) => {
-                 console.log("writeResult:", writeResult);
-             })
-             .catch((writeError) => {
-                 console.log("writeError occurred:", writeError);
-             })
+         async function insertPatient(collection, name, ssn, bloodType, medicalRecords, policyNumber, provider) {
```

Comment:
typically, if you have more than 4 parameters with a JS function, you want to switch to using object destructuring here. But that might make it more complicated.

Reply:
Agreed, and also identified when code style was linted with Airbnb rules. We had an internal discussion about this and some people preferred the signature that included each field explicitly rather than implicitly within a document. Since this function is specific to the fabricated example and not an exemplar on something specific to MongoDB (such as handling a connection lifecycle, I'll leave the parameters expanded if that's ok with you.